CHANGES.md

```diff
@@ -50,6 +50,8 @@ To be released.
     a certain amount of time.  [[#1741], [#1744]]
  -  (Libplanet.Net) `NetMQTransport` no longer attempts to retry failed
     communications.  [[#1751], [#1752]]
+ -  (Libplanet.Net) `Swarm<T>.PollBlocksAsync` now sync block like
+    `Swarm<T>.FillBlocksAsync`.  [[#1731], [#1759]]
```

Comment:
Since both `PollBlocksAsync()` & `FillBlocksAsync()` are internal methods, IMO it is not a good idea to mention them in the changelog.  Readers even don't get what it means.

How about adjusting it like below?

```suggestion
  - (Libplanet.Net) Fixed races between `Swarm<T>`'s dual systems for block
    synchronization (polling and event-driven in parallel) on updating chain by
    serializing the post-download process which is common for both
    synchronization mechanisms.  [[#1731], [#1759]]
```

Reply:
https://github.com/planetarium/libplanet/pull/1759/commits/8ae1b673d8cf52bc934b9cff406a42430b698985 amend it!